src/components/Navigator/QuickNavigationModal.vue

```diff
@@ -9,180 +9,387 @@
 -->
 
 <template>
-  <div class="quick-navigation-modal">
-    <div class="container">
+  <div
+    class="quick-navigation"
+    @keydown.down.exact.prevent="focusNext"
+    @keydown.esc.exact.prevent="closeQuickNavigationModal"
+    @keydown.up.exact.prevent="focusPrev"
+    @keydown.enter.exact="handleKeyEnter"
+  >
+    <div
+      class="quick-navigation__modal-shadow"
+      @click.self="closeQuickNavigationModal"
+    >
       <div
-        class="close-icon"
-        @click="closeQuickNavigationModal()"
+        class="quick-navigation__container"
       >
-        <p>
-          Close
-        </p>
-      </div>
-      <input
-        ref="input"
-        class="filter"
-        v-model="userInput"
-        type="text"
-      />
-      <div>
+        <FilterInput
+          v-model="userInput"
+          placeholder="Search symbols"
+          class="quick-navigation__filter"
+          focusInputWhenCreated
+          focusInputWhenEmpty
+          @input="focusedIndex = 0"
+        >
+          <template slot="icon">
+            <div
+              class="quick-navigation__magnifier-icon-container"
+              :class="{ 'blue': userInput.length }"
+            >
+              <MagnifierIcon />
+            </div>
+          </template>
+          <template #input-menu-items>
+            <div
+              v-if="!userInput.length"
+              class="quick-navigation__open-key-container"
+            >
+              <kbd>Cmd+Shift+o</kbd>
+            </div>
+          </template>
+        </FilterInput>
         <div
-          v-for="(symbol, idx) in filteredSymbols"
-          class="symbol-match"
-          @click="closeQuickNavigationModal()"
-          :key="idx"
+          class="quick-navigation__match-list"
+          :class="{ 'active' : debouncedInput.length }"
         >
-          <Reference :url="symbol.path" :id="idx">
-            <span>
-              -
-              {{ symbol.title.slice(0, symbol.start) }}
-            </span>
-            <QuickNavigationHighlighter
-              :text="symbol.substring"
-              :matcherText="debouncedInput"
-            />
-            <span>
-              {{ symbol.title.slice(symbol.start + symbol.matchLength) }}
-            </span>
-          </Reference>
+          <div
+            v-if="noResultsWereFound"
+            class="no-results"
+          >
+            <p>
+              No results found.
+          </p>
```

Comment:
```suggestion
            </p>
```

Reply:
Resolved in https://github.com/apple/swift-docc-render/pull/396/commits/62db4a637515e8fd67f83266a028ced48f825d01